homeassistant/components/roku/media_player.py

```diff
@@ -367,12 +375,13 @@ async def async_play_media(self, media_type: str, media_id: str, **kwargs) -> No
         """Tune to channel."""
         extra: dict[str, Any] = kwargs.get(ATTR_MEDIA_EXTRA) or {}
 
-        if media_type not in (MEDIA_TYPE_APP, MEDIA_TYPE_CHANNEL):
+        if media_type not in (MEDIA_TYPE_APP, MEDIA_TYPE_CHANNEL, MEDIA_TYPE_URL):
```

Comment:
It probably doesn't matter here but you could make this a constant and then unpack it below with a `*`

Reply:
will address in other PR